lib-manual/helpers/webhook_verifier.rb

```diff
@@ -0,0 +1,74 @@
+require 'openssl'
+require 'securecompare'
+
+module MuxRuby
+  module Helpers
+    class WebhookVerifier
+      DEFAULT_TOLERANCE = 300
+      HEADER_SCHEMES = [:v1].freeze
+
+      # Initializer.
+      #
+      # @param [String] secret the webhook secret from your Mux control panel
+      # @param [Integer] tolerance the signature timing tolerance, in seconds (generally leave as is)
+      # @param [Array<Symbol>] header_schemes the list of accepted header schemes for this verifier
+      def initialize(secret: nil, tolerance: DEFAULT_TOLERANCE, header_schemes: [:v1])
+        raise "secret '#{secret.inspect}' must be a String" unless secret.is_a?(String)
+        raise "tolerance '#{tolerance.inspect}' must be a positive number." \
```

Comment:
I'm not sure we use `tolerance` anywhere else, typically it's `expiration`. Why the change here?

Reply:
It's tolerance in the node SDK.